Dolapo47/Transport-App
# Reset password

### What does this PR do?
#### A summary of what this task is about
This pull request creates the make an admin route

### Description of Task to be completed?
#### A list of the major tasks you did to complete this feature/bug/chore
set up the mailing service
set up the create admin service
set up the create admin route

### How should this be manually tested?
Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
register a new user on postman; retrieve the token from registered email; verify the email in the verify password route; then as an admin update a user to an admin
